_utility/send_gh_comment.sh

```diff
@@ -15,44 +15,49 @@ source "$DIR/send_gh_comment_validation.sh"
 COMMENT="Deployed: $DEPLOYMENT_URL"
 
 function get_pr_number_by_branch_name() {
-  PR_NUMBER=$(curl -X GET -u ${GITHUB_TOKEN}:x-oauth-basic "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/pulls?head=${REPO_OWNER}:${CIRCLE_BRANCH}" | jq ".[0].number")
-	if [ -z "$PR_NUMBER" ]; then
-		echo "PR_NUMBER not specified, and can't be determined in the current context"
-		exit 1
-	fi
+  if [ -n "$CIRCLE_BRANCH" ]; then
+    pr_number_by_branch=$(curl -X GET -u ${GITHUB_TOKEN}:x-oauth-basic "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/pulls?head=${REPO_OWNER}:${CIRCLE_BRANCH}" | jq ".[0].number")
+    if [ $? -ne 0 ] && [ "$pr_number_by_branch" != "null" ]; then
+      PR_NUMBER=pr_number_by_branch
+    fi
+  fi
 }
 
 function get_last_comment_id_generated_by_bot() {
   LAST_COMMENT_ID=$(curl https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/issues/$PR_NUMBER/comments?access_token=$GITHUB_TOKEN -X GET | jq "[.[] | select(.user.login==\"$GITHUB_USERNAME\")][0].id")
   if [ $? -ne 0 ]; then
-    echo "Error while parsing last commend id"
-    exit 1
+    echo "null"
   fi
   echo $LAST_COMMENT_ID
 }
 
 function update_last_comment() {
-	echo "Found a previous comment with id:$LAST_COMMENT_ID . Updating comment"
-	if [ "$EXECUTE" == "1" ]; then
-		curl https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/issues/comments/$LAST_COMMENT_ID?access_token=$GITHUB_TOKEN \
-			-H "Content-Type: application/json" \
-			-X POST \
-			-d "{ \"body\":\"$COMMENT\" }"
-	fi
+  echo "Found a previous comment with id:$LAST_COMMENT_ID . Updating comment"
+  if [ "$EXECUTE" == "1" ]; then
+    curl https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/issues/comments/$LAST_COMMENT_ID?access_token=$GITHUB_TOKEN \
+      -H "Content-Type: application/json" \
+      -X POST \
+      -d "{ \"body\":\"$COMMENT\" }"
+  fi
 }
 
 function create_comment() {
-	echo "Creating a comment with deployment url"
-	if [ "$EXECUTE" == "1" ]; then
-		curl https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/issues/$PR_NUMBER/comments?access_token=$GITHUB_TOKEN \
-			-H "Content-Type: application/json" \
-			-X POST \
-			-d "{ \"body\":\"$COMMENT\" }"
-	fi
+  echo "Creating a comment with deployment url"
+  if [ "$EXECUTE" == "1" ]; then
```

Comment:
I don't know, in bash is pretty standard to do this because there are no booleans.